docling/utils/glm_utils.py

```diff
@@ -304,6 +304,15 @@ def to_docling_document(doc_glm, update_name_label=False) -> DoclingDocument:
                 current_list = None
 
                 doc.add_heading(text=text, prov=prov)
+            elif label in [DocItemLabel.PAGE_HEADER, DocItemLabel.PAGE_FOOTER]:
+                current_list = None
+
+                doc.add_text(
+                    label=DocItemLabel(name_label),
+                    text=text,
+                    prov=prov,
+                    parent=doc.furniture,
```

Comment:
that does not look right, we should have it in the body referenced to the furniture.

Reply:
That looks absolutely right in the original intention. Furniture and body were supposed to be disjoint structures with no references between.